cookbooks/ambari/recipes/ambari_views_setup.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Cookbook Name:: ambari
+# Recipe:: ambari_views_setup
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+
+http_request 'Polling Ambari WebUi' do
+  url "#{node['ambari']['ambari_server_base_url']}"
+  retries 24
+  retry_delay 5
+  # notifies :run, 'bash[create_instance_of_Files_view]', :immediately
+end
+
+# create an instance of Files View
+bash 'create_instance_of_Files_view' do
```

Comment:
Is this REST call idempotent?

Reply:
This is kinda idempotent as the first call create the resource but later calls are rejected as the resource is already created.